WeWatch/WeWatch/UICommons/Pill.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Pill.swift
+//  WeWatch
+//
+//  Created by Anton on 26/12/2024.
+//
+
+import SwiftUI
+
+struct Pill: View {
+    @State private var didTap: Bool = false
+    private var pillTitle: String = .init()
+    var body: some View {
+        Button(action: {
+            self.didTap = true
+        }) {
+            Text(pillTitle)
+        }
+        .padding(.top, 2)
```

Comment:
- For vertical use `.padding(.vertical, 2)`
- For horizontal `.padding(.horizontal, 16)